Polyhedron/demo/Polyhedron/Scene_polyhedron_item.h

```diff
@@ -98,6 +99,9 @@ class SCENE_POLYHEDRON_ITEM_EXPORT Scene_polyhedron_item
     std::vector<QColor>& color_vector();
     void set_color_vector_read_only(bool on_off);
     bool is_color_vector_read_only();
+
+    void set_patch_id(Polyhedron::Face_handle f,int i) const;
```

Comment:
The basic design of the item polyhedron demo, at the beginning was to avoid have the CGAL API exposed in the items API. Why do you need `set_patch_id` at the level of the item?!